examples/gitlab/build-and-deploy-mender-artifact.gitlab-ci.yml

```
@@ -0,0 +1,71 @@
+#
+# The example shows how to build, upload and deploy a Mender Artifact.
+#
+# 'MENDER_SERVER_ACCESS_TOKEN' secret is required to be set in GitLab repo settings
+#
+
+stages:
+  - build
+  - publish
+  - deploy
+
+include:
+  - project: 'Northern.tech/Mender/mender-ci-workflows'
+    file: 'templates/gitlab/mender-artifact-build.gitlab-ci.yml'
+    ref: 'master'
+  - project: 'Northern.tech/Mender/mender-ci-workflows'
+    file: 'templates/gitlab/mender-artifact-upload.gitlab-ci.yml'
+    ref: 'master'
+  - project: 'Northern.tech/Mender/mender-ci-workflows'
+    file: 'templates/gitlab/mender-deployment-create.gitlab-ci.yml'
+    ref: 'master'
+
+variables:
+  MENDER_ARTIFACT_NAME: artifact.mender       # Mender Artifact file name in the `MENDER_ARTIFACTS_FOLDER` folder
+  MENDER_RELEASE_NAME:  updated-auth_keys-1.0 # The name of Mender release
+
+#
+# Builds Mender artifact using 'mender-artifact' tool and upload it to MENDER_ARTIFACTS_FOLDER.
+#
+build:mender-artifact:
```

Please let's name the jobs in the examples pipeline same as the templates
```suggestion
mender:build:artifact:
```

---

(for all jobs and all examples)

---

I like `build:mender-artifact` here way more, but won't resist...